inmaputil/cmd.go

```diff
@@ -441,6 +441,29 @@ func init() {
 			defaultVal: "${GOPATH}/src/github.com/spatialmodel/inmap/inmap/testdata/preproc/gc_output.[DATE].nc",
 			flagsets:   []*pflag.FlagSet{preprocCmd.Flags()},
 		},
+		{
+			name: "Preproc.GEOSChem.chemFileStr",
+			usage: `
+              Preproc.GEOSChem.chemFileStr is the time interval of each file.
+              E.g. "3h" for 3 hours`,
+			defaultVal: "${GOPATH}/src/github.com/spatialmodel/inmap/inmap/testdata/preproc/gc_output.[DATE].nc",
+			flagsets:   []*pflag.FlagSet{preprocCmd.Flags()},
+		},
+		{
+			name: "Preproc.GEOSChem.chemRecordStr",
+			usage: `
+              Preproc.GEOSChem.chemRecordStr is the string specifying the time interval between different records.
```

Comment:
Preproc.GEOSChem.ChemRecordInterval specifies the time duration represented by each GEOS-Chem output record.